integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseUtils.java

```diff
@@ -56,7 +56,8 @@ public static long getGid(String userName) {
    * @return user name
    */
   public static String getUserName(long uid) throws IOException {
-    return ShellUtils.execCommand("id", "-nu", Long.toString(uid)).trim();
+    // Simply use the 'getent' command to get username from uid and cut the first filed delimited by colon
+    return ShellUtils.execCommand("getent", "passwd", Long.toString(uid)).trim().split(":")[0];
```

Comment:
different from id, getent may not be available on different platforms, e.g. it is unavailable on MacOS. Can we make it platform dependent?

Reply:
yup, since 1.6 we added support of MacOS to FUSE.

Reply:
thanks for correcting, command 'id' works for MacOS and Linux 7 while it doesn't support for Linux 6. Linux OS dependency was added for command 'getent', any advise/guidance would be greatly appreciated!